README.md

```diff
@@ -2,7 +2,26 @@
 
 # Composer
 
-## Changesets
+This is a monorepo that supports components, analytics, 3rd party integrations (such as contentful), payment methods, messaging and much more.
```

Comment:
We do not support payment methods 😉

Reply:
And we have a better definition of composer _somewhere_, I will search for it and share it with you